tests/core/pfline/test_pfline_arithmatic_kind_and_error.py

```diff
@@ -5,21 +5,50 @@
 from functools import lru_cache
 from typing import Any, Dict, Iterable
 
+import numpy as np
 import pandas as pd
 import pytest
 from utils import id_fn  # relative to /tests
 
 import portfolyo as pf
 from portfolyo import Q_, PfLine
 from portfolyo.core.pfline import Kind, Structure, arithmatic, create
+from portfolyo import tools
 
 # TODO: various timezones
 
 # TODO: use/change STRICT setting
 
+# NAMES_AND_UNITS = {
+#     "w": "pint[MW]",
+#     "q": "pint[MWh]",
+#     "p": "pint[Eur/MWh]",
+#     "r": "pint[Eur]",
+#     "duration": "pint[h]",
+#     "t": "pint[degC]",
+#     "nodim": "pint[dimensionless]",
+# }
+
+
+def get_value(
```

Comment:
Is there a reason why this function is defined here, instead of changing and using the function in `dev`?

Reply:
not really, changed the location